jenkins-simulate-data.sh

```diff
@@ -63,24 +63,60 @@ echo "Leaf node set to: $leafNode"
 # try to trick pip into working in the WORKSPACE
 #
 export HOME="${WORKSPACE:-$HOME}"
-/usr/bin/pip3 install cdislogging
-/usr/bin/pip3 install --user -r requirements.txt
-#python setup.py develop --user
+if [ -f ./pyproject.toml ]; then
```

Comment:
Wouldn't it be better to create a whl / egg file in data-simulator using poetry and use that here as a library?